### PR TITLE
test: speed up seeding when testing

### PR DESCRIPTION
**Pull request by ojeytonwilliams**
- test: speed up seeding
- fix: make seed-demo-user reject bad arguments

Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

`pnpm run seed` triggers a bunch of other scripts that it's pointless to run while testing. They only need to be run once before testing, and this happens when the client is started.

### CODE REVIEW

File: tools/scripts/seed/seed-demo-user.js [hunk 1]
```diff
@@ -186,9 +186,13 @@ const run = async () => {
   if (process.argv[2] === 'certified-user') {
     await user.insertOne(fullyCertifiedUser);
     await user.insertOne(blankUser);
-  } else {
+  } else if (!process.argv[2]) {
     await user.insertOne(demoUser);
     await user.insertOne(blankUser);
+  } else {
+    throw new Error(
+      'Invalid argument - This can only be called with zero or one argument. If there is an argument, it must be "certified-user"'
```
Comment (ShaunSHamilton) [inline tools/scripts/seed/seed-demo-user.js:194]: How long where you stuck on this?

Comment (ojeytonwilliams) [reply tools/scripts/seed/seed-demo-user.js:194]: I just wanted this to fail noisily if we changed the arguments it accepted.

File: cypress.config.js [hunk 2]
```diff
@@ -2,6 +2,10 @@ const { execSync } = require('child_process');
 const { existsSync } = require('fs');
 const { defineConfig } = require('cypress');
 
+function seed(args = []) {
+  execSync('node tools/scripts/seed/seed-demo-user' + args.join(' '));
```
[the remark this answers is not in the document]
Comment (ojeytonwilliams) [reply cypress.config.js]: Yeah, thanks. I'll fix it.

**Review by ShaunSHamilton — APPROVED** [review]
LGTM 👍

**Review by huyenltnguyen — APPROVED** [review]
LGTM

**Pull request merged**